CHANGELOG.md

```diff
@@ -6,6 +6,16 @@
 
 ### Changes
 
+  * [#769](https://github.com/toptal/chewy/pull/769): **(Breaking)** Remove `Chewy::Index.index_params`, use `Chewy::Index.specification_hash` instead ([@Vitalina-Vakulchyk][])
+  * [#769](https://github.com/toptal/chewy/pull/769): **(Breaking)** Remove `Chewy::Index.derivable_index_name`, use `Chewy::Index.derivable_name` instead ([@Vitalina-Vakulchyk][])
+  * [#769](https://github.com/toptal/chewy/pull/769): **(Breaking)** Remove `Chewy::Index.build_index_name`, use `Chewy::Index.index_name` instead ([@Vitalina-Vakulchyk][])
+  * [#769](https://github.com/toptal/chewy/pull/769): **(Breaking)** Remove `Chewy::RakeHelper.reset_index`, use `Chewy::RakeHelper.reset` instead ([@Vitalina-Vakulchyk][])
+  * [#769](https://github.com/toptal/chewy/pull/769): **(Breaking)** Remove `Chewy::RakeHelper.reset_all`, use `Chewy::RakeHelper.reset` instead ([@Vitalina-Vakulchyk][])
+  * [#769](https://github.com/toptal/chewy/pull/769): **(Breaking)** Remove `Chewy::RakeHelper.update_index`, use `Chewy::RakeHelper.update` instead ([@Vitalina-Vakulchyk][])
+  * [#769](https://github.com/toptal/chewy/pull/769): **(Breaking)** Remove `Chewy::RakeHelper.update_all`, use `Chewy::RakeHelper.update` instead ([@Vitalina-Vakulchyk][])
+  * [#769](https://github.com/toptal/chewy/pull/769): **(Breaking)** Remove `rake chewy:apply_changes_from`, use `rake chewy:journal:apply` instead ([@Vitalina-Vakulchyk][])
+  * [#769](https://github.com/toptal/chewy/pull/769): **(Breaking)** Remove `rake chewy:clean_journal`, use `rake chewy:journal:clean` instead ([@Vitalina-Vakulchyk][])
+
```

Comment:
```suggestion
  * [#769](https://github.com/toptal/chewy/pull/769): **(Breaking)** Removal of deprecated methods and rake tasks  ([@Vitalina-Vakulchyk][]):
    * `Chewy::Index.index_params` is removed, use `Chewy::Index.specification_hash` instead
    * `Chewy::Index.derivable_index_name` is removed, use `Chewy::Index.derivable_name` instead
    * `Chewy::Index.default_prefix` is removed, use `Chewy::Index.prefix` instead
    * `Chewy::Index.build_index_name` is removed, use `Chewy::Index.index_name` instead
    * `Chewy::RakeHelper.reset_index` is removed, use `Chewy::RakeHelper.reset` instead
    * `Chewy::RakeHelper.reset_all` is removed, use `Chewy::RakeHelper.reset` instead
    * `Chewy::RakeHelper.update_index` is removed, use `Chewy::RakeHelper.update` instead
    * `Chewy::RakeHelper.update_all` is removed, use `Chewy::RakeHelper.update` instead
    * `rake chewy:apply_changes_from` is removed, use `rake chewy:journal:apply` instead
    * `rake chewy:clean_journal` is removed, use `rake chewy:journal:clean` instead
```

Reply:
Probably something like this should be added to the upgrade/migration notes as well.

Reply:
> Probably something like this should be added to the upgrade/migration notes as well.

Actually, they all have been deprecated since 2017. I think it should be safe just to drop them finally, with a note in the CHANGELOG only.

Reply:
Currently, I have just updated CHANLOG again.
I can do more changes if needed.